curation_concerns-models/app/models/concerns/curation_concerns/upload_set_behavior.rb

```diff
@@ -7,7 +7,6 @@ module UploadSetBehavior
     included do
       has_many :file_sets, predicate: ActiveFedora::RDF::Fcrepo::RelsExt.isPartOf
 
-      property :creator, predicate: ::RDF::DC.creator
```

Comment:
Sufia 6 is getting by just fine without, so I'd suggest not letting this PR get blocked.